src/net.cpp

```diff
@@ -468,6 +468,10 @@ void CConnman::DumpBanlist()
         SetBannedSetDirty(false);
     }
 
+    // update UI
```

Comment:
I think an appropriate place for this would be in `SweepBanned()` itself, as the list is changed there. It just happens to be called from here, but there are other places.

Reply:
Agree.